java/pom.xml

```diff
@@ -46,6 +46,7 @@
     <errorprone.javac.version>9+181-r4173-1</errorprone.javac.version>
     <error_prone_core.version>2.16</error_prone_core.version>
     <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
+    <checker.framework.version>3.38.0</checker.framework.version>
```

Comment:
v3.39.0 just released a few days ago, which adds additional support for Java 21 compilation